src/injector/inject.ts

```diff
@@ -63,8 +85,11 @@ export function trap () {
                 return oldBind.apply(this, args);
             } else if (
                 args[0] &&
-                Object.prototype.hasOwnProperty.call(args[0], "editingTarget") &&
-                Object.prototype.hasOwnProperty.call(args[0], "runtime")
+                Object.prototype.hasOwnProperty.call(
+                    args[0],
```

Comment:
unnecessary line breaks!!!!!!

Reply:
Prettier says it is necessary.

Reply:
It's unnecessary. It’s neither pretty nor does it improve readability much